### PR TITLE
adding static nginx website

### PR DESCRIPTION
**Pull request by venturel**

### CODE REVIEW

File: manifests/static.pp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  $ssl_dh_param               = undef,
+  $basic_authentication_file  = undef,
+  $static_ssl_out              = false
+  ){
+
+  if ! defined(Class['nginx']) {
+    fail('You must include the nginx base class before using any nginx defined resources')
+  }
+
+  $url = "http://${hostname}"
```
Comment (MattiasGees) [inline manifests/static.pp]: Where is this used?

File: manifests/static.pp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  if($ssl_certificate == undef or $ssl_certificate_key == undef){
+    $ssl = false
+  } else {
+    $ssl = true
+  }
+
+  if($static_ssl_out){
+     $_protocol = 'https'
```
Comment (MattiasGees) [inline manifests/static.pp]: Where is this used?

File: manifests/static.pp [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+# == Class nginx::static
+#
+# This class will install a static
+#
+define nginx::static (
+  $root_path                  = undef,
+  $option_paths                = undef,
+  $hostname                   = undef,
+  $server_aliases             = [],
+  $static_host                 = undef,
```
Comment (MattiasGees) [inline manifests/static.pp]: I this used?

File: manifests/static.pp [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  $port                       = 80,
+  $ip                         = '',
+  $client_max_body_size       = '1m',
+  $ssl_certificate            = undef,
+  $ssl_certificate_key        = undef,
+  $ssl_dh_param               = undef,
+  $basic_authentication_file  = undef,
```
Comment (MattiasGees) [inline manifests/static.pp]: Is this used?

File: manifests/static.pp [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  $static_ssl_out              = false
```
Comment (MattiasGees) [inline manifests/static.pp]: Is this used?

**Comment by MattiasGees** [comment]
Except the 3 vars that I don't see used I give this a 👍

File: templates/etc/nginx/sites-available/static.conf.erb [hunk 6]
```diff
@@ -0,0 +1,77 @@
+# {{ ansible_managed }}
```
Comment (MattiasGees) [inline templates/etc/nginx/sites-available/static.conf.erb]: Remove this maybe :)

**Comment by SamClinckspoor** [comment]
👍 :shipit:

**Pull request merged**